evaluate.sh

```diff
@@ -36,6 +36,7 @@ RESET='\033[0m' # No Color
 MEASUREMENTS_FILE="measurements_1B.txt"
 RUNS=5
 DEFAULT_JAVA_VERSION="21.0.1-open"
+BUILD_JAVA_VERSION="21.0.1-open"
```

Comment:
```suggestion
: "${BUILD_JAVA_VERSION:=21.0.1-open}"
```

That way, it can be overridden when launching:

```
BUILD_JAVA_VERSION=22.ea.31-open ./evaluate.sh linl33
```